src/applications/lgy/coe/containers/App.jsx

```diff
@@ -1,12 +1,40 @@
 import React from 'react';
-
+import { connect } from 'react-redux';
+import { isLoggedIn } from 'platform/user/selectors';
 import RoutedSavableApp from 'platform/forms/save-in-progress/RoutedSavableApp';
 import formConfig from '../config/form';
 
-export default function App({ location, children }) {
+const FORM_PATHS = [
+  'applicant-information-summary',
+  'applicant-contact-information',
+  'applicant-communication-preference',
+  'service-status',
+  'service-history',
+  'service-decoration',
+  'existing-loan-screener',
+  'loan-intent',
+  'loan-history',
+  'document-upload-summary',
+  'upload-supporting-documents',
+  'review-and-submit',
+  'confirmation',
+];
```

Comment:
Wouldn't it be better to get the `FORM_PATHS` from the `formConfig` chapter paths?